packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -437,6 +467,61 @@ const hideVideoLoadingSpinner = async (page: Page): Promise<void> => {
   await page.addStyleTag({ content: '[data-ui-id="stream-media-loading-spinner"] {display: none}' });
 };
 
+const MASK_ATTRIB_KEY = 'data-ui-id';
+const MASK_ATTRIB_VALUE = 'stream-media-video-mask';
+const MASK_SELECTOR = `[${MASK_ATTRIB_KEY}="${MASK_ATTRIB_VALUE}"]`;
+/**
+ * Masks the video element in {@link VideoTile} by overlaying an opaque <div/>.
+ *
+ * This function is destructive: It changes the video tile's DOM irreversibly.
+ * This function is idempotent: It is safe to call this function multiple times.
+ */
+async function maskVideos(page: Page): Promise<void> {
```

Comment:
note: we should only need to do this for live tests, the hermetic tests (I think) all have fake video DOM already

Reply:
Yeah, I noticed the unnecessary snapshot updates for the hermetic tests.

I'm undecided on whether I prefer to have uniform snapshots and code (one less difference to learn between live vs hermetic) or different snapshots.

Reply:
On the balance, I think the consistency between hermetic / live tests beats an extra DOM node during testing.